cf-serverd/server.c

```diff
@@ -241,7 +241,7 @@ static void SpawnConnection(EvalContext *ctx, const char *ipaddr, ConnectionInfo
         Log(LOG_LEVEL_ERR,
             "Unable to spawn worker thread. (pthread_create: %s)",
             GetErrorStr());
-        goto cleanup;
+        /* Execute cleanup: which is the next instruction. */
```

Comment:
Make sense. Will revert this. I've just seen some unnecessary code which could be removed. In fact this whole function can be refactored easily to get rid of goto.